src/components/templates/TemplateSpotDetail.vue

```diff
@@ -61,88 +61,157 @@
-                            </tr>
-                            <tr v-if="ownerInfoDetails.City">
-                                <td>City</td>
-                                <td>{{ ownerInfoDetails.City }}</td>
-                            </tr>
-                            <tr v-if="ownerInfoDetails.EmailID">
-                                <td>EmailID</td>
-                                <td>{{ ownerInfoDetails.EmailID }}</td>
-                            </tr>
-                            <tr v-if="ownerInfoDetails.KYCStatus">
-                                <td>KYCStatus</td>
-                                <td>
-                                    {{
-                                        getKYCStatus(ownerInfoDetails.KYCStatus)
-                                    }}
-                                </td>
-                            </tr>
-                            <tr v-if="spotDetails.UpdatedAt">
-                                <td>Last Call Date</td>
-                                <td>
-                                    <AtomDatePicker
-                                        :assignedDate="spotDetails.LastCallDate"
-                                        :size="'is-small'"
-                                        @changed="changeLastCallDate"
-                                    >
-                                    </AtomDatePicker>
-                                </td>
-                            </tr>
-                            <tr>
-                                <td>Remark</td>
-                                <td>
-                                    <AtomTextarea
-                                        v-model="spotDetails.Remark"
-                                        :rowNo="2"
-                                        @changed="changeRemark"
-                                    />
-                                </td>
-                            </tr>
-                            <tr v-if="paymentDetails">
-                                <td>Account</td>
-                                <td>{{ paymentDetails }}</td>
-                            </tr>
-                        </table>
-                    </div>
-                    <div class="goto-btn">
-                        <AtomButton
-                            @click.native="
-                                goToInterestedVO([
-                                    spotDetails.Lat,
-                                    spotDetails.Long,
-                                ])
-                            "
-                        >
-                            Interested VO's
-                        </AtomButton>
-                    </div>
-                    <div class="goto-btn" v-if="isAvailable">
-                        <AtomButton @click.native="changeAvailability(-1)">
-                            Mark Rented
-                        </AtomButton>
+            <div class="only-to-agent-admin">
```

Comment:
you can keep the class name as `only-to-agent` because anything visible to agent would be definition be visible to admin